tox.ini

```diff
@@ -42,7 +42,9 @@ commands =
 [testenv:checkqa]
 basepython = python3.6
 commands =
-    flake8 --inline-quotes "double" pinax
+    flake8 pinax
+    isort pinax/**/*.py -c
 deps =
     flake8 == 2.5.4
     flake8-quotes == 0.11.0
+    isort == 4.2.15
```

Comment:
Deps could be centralized as an `extras_require` for e.g. `qa` in setup.py.

Reply:
See https://github.com/Vimjas/covimerage/blob/c8116278c3b00e80c9be6c3e30abe9ef6dab23c6/setup.py#L94-L106.